utopia-remix/app/routes/projects.tsx

```diff
@@ -70,76 +112,302 @@ const ProjectsPage = React.memo(() => {
-        <button
-          className={button({ color: 'accent', size: 'medium' })}
-          onClick={loadMore(projects.length)}
+      <div
+        style={{
+          display: 'flex',
+          flexGrow: 1,
+          flexDirection: 'column',
+          gap: marginSize,
+        }}
+      >
+        <div
+          onMouseDown={clearSelectedProject}
+          style={{
+            height: 60,
+            flex: 0,
+            display: 'flex',
+            flexDirection: 'row',
+            gap: 15,
+          }}
+        >
+          {newProjectButtons.map((p) => (
+            <button className={newProjectButton({ color: p.color })} onClick={p.onClick}>
+              <span>{p.title}</span>
+            </button>
+          ))}
+        </div>
+        <div
+          onMouseDown={clearSelectedProject}
+          style={{ fontSize: 16, fontWeight: 600, padding: '5px 10px', userSelect: 'none' }}
         >
-          Load more
-        </button>
-      ) : null}
+          {searchQuery !== '' ? (
+            <span>
+              <span style={{ color: 'gray', paddingRight: 3 }}>
+                <span
+                  onClick={() => {
+                    setSearchValue('') // Clear the search value state
+                    setFilteredProjects(data.projects) // Reset the filtered projects
+                    setSearchQuery('') // Reset the search query
+                    const inputElement = document.getElementById('search-input') as HTMLInputElement
+                    if (inputElement) {
+                      inputElement.value = '' // Clear the input field value
+                    }
+                  }}
+                  style={{ cursor: 'pointer' }}
+                >
+                  ←{' '}
+                </span>{' '}
+                Search results for
+              </span>
+              <span> "{searchQuery}"</span>
+            </span>
+          ) : (
+            selectedCategory
+          )}
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            flexWrap: 'wrap',
+            alignContent: 'flex-start',
+            gap: marginSize,
+            flexGrow: 1,
+            flexDirection: 'row',
+            overflowY: 'scroll',
+            scrollbarColor: 'lightgrey transparent',
+          }}
+        >
+          {filteredProjects.map((project) => (
+            <ProjectCard
+              key={project.proj_id}
+              project={project as Project}
+              selected={project.proj_id === selectedProject.selectedProjectId}
+              onSelect={() => handleProjectSelect(project.proj_id)}
+            />
+          ))}
+          {!reachedEnd ? (
+            <button onClick={loadMore(filteredProjects.length)} style={{ width: 200, height: 60 }}>
+              Load More
+            </button>
+          ) : null}
+        </div>
+      </div>
     </div>
   )
 })
 ProjectsPage.displayName = 'ProjectsPage'
 
 export default ProjectsPage
+
+type Project = {
```

Comment:
how come instead using the `Project` type you get from Prisma? (heads up! in https://github.com/concrete-utopia/utopia/pull/4862 there's a new type called `ProjectWithoutContent` that would probably be even better!)